ui/src/dashboards/graphics/graph.js

```diff
@@ -0,0 +1,266 @@
+import React from 'react'
+
+export const graphTypes = [
```

Comment:
kewl. these look great. i wish all graphs and single stats looked like these, actually!